config/database.yml

```diff
@@ -1,25 +1,19 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: 5
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/development
```

Comment:
Ooh interesting! Is this valid? I usually have a name like `overwatch_team_comp_dev` here, then I'd have to `createdb overwatch_team_comp_dev` before running migrations. Maybe I've been making local PostgreSQL Rails development harder than necessary.